pkg/registry/apis/wireset.go

```diff
@@ -10,13 +10,26 @@ import (
 	"github.com/grafana/grafana/pkg/registry/apis/folders"
 	"github.com/grafana/grafana/pkg/registry/apis/iam"
 	"github.com/grafana/grafana/pkg/registry/apis/provisioning"
+	"github.com/grafana/grafana/pkg/registry/apis/provisioning/webhooks"
 	"github.com/grafana/grafana/pkg/registry/apis/query"
 	"github.com/grafana/grafana/pkg/registry/apis/secret"
 	"github.com/grafana/grafana/pkg/registry/apis/service"
 	"github.com/grafana/grafana/pkg/registry/apis/userstorage"
 	"github.com/grafana/grafana/pkg/services/pluginsintegration/plugincontext"
 )
 
+// HACK: This is a hack so that wire can uniquely identify dependencies
```

Comment:
Notice this thing. I couldn't use an Extra or ExtraBuilder directly because wire would consider them duplicates and it didn't know what to do. I am not sure if there is a better way to create a list of "Features"

Reply:
for other features we have a single OSS and Enterprise implementation; i'm not sure if that will make more sense long term or this approach.  

EG: do we wire in an array of things, or does the enterprise instance know the set of things it supports